docs/Manuals/client-manual.rst

```diff
@@ -29,34 +32,44 @@ files. Double-clicking ``freeciv21-client`` should start it up.
 
 .. _Start Screen:
 .. figure:: /_static/images/gui-elements/start-screen.png
-    :scale: 75%
+    :scale: 70%
     :align: center
     :alt: Freeciv21 start screen
     :figclass: align-center
 
     Start Screen with NightStalker Theme
 
 
+.. note::
+  You may have noticed that the version string is a bit different than other games or software. The version
+  number for Freeciv21 is broken into a few parts:
+  ``[major version].[minor version].[hash version].[version string level]-[version string]``. In
```

Comment:
I am not a fan of how the version is done either, just trying to help break out the parts. I would like to see the `hash version` support hex so we can just grab the last 6 chars of the gh hash instead of having to turn it into a number. #1742